Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -641,6 +679,20 @@ UIView SetupButtons(nfloat width, nfloat height)
 			return null;
 		}
 
+		//Should this really be this cheaty? Or is there a better way to obtain the native more icon?
+		//Note: Using icons improperly may violate iOS Human Interface Guidelines and cause the app to be rejected from App Store
+		UIImage GetMoreImage()
+		{
+			var tabBar = new UITabBar();
+			var item = new UITabBarItem(UITabBarSystemItem.More, 0);
+			tabBar.Items = new[] { item };
+
+			var itemView = tabBar.Subviews[0];
+			var imageView = itemView.Subviews[0] as UIImageView;
+
+			return imageView?.Image.ImageWithRenderingMode(UIImageRenderingMode.AlwaysTemplate);
```

Comment:
Also, once you've retrieved the image you should cache it so every cell doesn't have to do this.